src/DataWrangling/metadata_field.jl

```diff
@@ -17,11 +17,16 @@ end
 
 function native_grid(metadata::Metadata, arch=CPU(); halo = (3, 3, 3))
     Nx, Ny, Nz = size(metadata)
+    if is_three_dimensional(metadata)
+        z = z_interfaces(metadata)
+    else
+        z = (0,1)
```

Comment:
```suggestion
        z = (0, 1)
```

but why this choice? seems ad hoc?

Reply:
I think in the end the bottom height is set onto a `Field{Center, Center, Nothing}` so it's completely arbitrary

Reply:
Wondering whether we should use a Flat z dimension for the bathymetry's native grid. Would that work with interpolating onto a grid with finite Nz? @simone-silvestri?